packages/amplify_api/lib/method_channel_api.dart

```diff
@@ -33,8 +34,13 @@ class AmplifyAPIMethodChannel extends AmplifyAPI {
     try {
       return await _channel.invokeMethod('addPlugin');
     } on PlatformException catch (e) {
-      throw AmplifyException("API plugin has already been added, " +
-          "multiple plugins for API category are currently not supported.");
+      if (e.code == "AmplifyAlreadyConfiguredException") {
+        throw AmplifyAlreadyConfiguredException(AmplifyExceptionMessages.alreadyConfiguredDefaultMessage,
```

Comment:
Nit: would it be better to format the code as: 

```
        throw AmplifyAlreadyConfiguredException(
             AmplifyExceptionMessages.alreadyConfiguredDefaultMessage,
             recoverySuggestion: AmplifyExceptionMessages.alreadyConfiguredDefaultSuggestion
       );
```

To make it easier to read?

Reply:
Made this change to all relevant files.